File: sistema_facturacion_api/sistema_facturacion_api.Data/DTOs/NuevaFacturaDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace sistema_facturacion_api.Data.DTOs
{
    public class NuevaFacturaDTO
    {
        public TblFacturasDTO Factura { get; set; }
        public List<TblDetalleDeFacturasDTO> DetalleFactura { get; set; }
    }
}

File: sistema-facturacion-front/src/app/models/dtos/nueva-factura-dto.ts
import { TblDetalleDeFacturasDTO } from "./tbl-detalle-de-facturas-dto";
import { TblFacturasDTO } from "./tbl-facturas-dto";


export class NuevaFacturaDTO{
  factura: TblFacturasDTO;
  detalleFactura: Array<TblDetalleDeFacturasDTO>;

  constructor(){
    this.factura = new TblFacturasDTO();
    this.detalleFactura = new Array<TblDetalleDeFacturasDTO>();
  }
}
